File: hw/jacome/U3/HW19AbstractFactory/JS/controller/WinFactory.js
const GUIFactory = require('./GUIFactory');
const WinButton = require('../view/WinButton');
const WinMenu = require('../model/WinMenu');

class WinFactory extends GUIFactory {
    createButton() {
        return new WinButton();
    }

    createMenu() {
        return new WinMenu();
    }
}

module.exports = WinFactory;

File: hw/paillacho/U3/HW19AbstractFactory/AbstractFactory/src/main/java/ec/edu/espe/abstractfactory/controller/WinFactory.java

package ec.edu.espe.abstractfactory.controller;

import ec.edu.espe.abstractfactory.model.Button;
import ec.edu.espe.abstractfactory.model.Menu;
import ec.edu.espe.abstractfactory.model.WinMenu;
import ec.edu.espe.abstractfactory.view.WinButton;

/**
 *
 * @author Carlos Paillacho
 */
public class WinFactory extends GUIFactory{

    @Override
    public Button createButton() {
        return (new WinButton());
    }

    @Override
    public Menu createMenu() {
        return (new WinMenu());

    }
}
